wasm-test/src/lib.rs

```diff
@@ -53,7 +54,9 @@ pub extern "C" fn check_commit_entry(encoded_allocation_of_input: u32) -> u32 {
     }
     let input: CommitInputStruct = result.unwrap();
 
-    let res = hdk::commit_entry(&input.entry_type_name, &input.entry_content);
+    let res = hdk::commit_entry(&input.entry_type_name, json!({
+        "entry_content": &input.entry_content
```

Comment:
You shoudn't need to write "entry_content".
Why not do directly: `json!(&input.entry_content)` ?